src/Countries/Greece.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Greece extends Country
+{
+    public function countryCode(): string
+    {
+        return 'el';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Πρωτοχρονιά'         => '01-01',
+            'Θεοφάνια'            => '01-06',
+            '25η Μαρτίου'         => '03-25',
+            'Πρωτομαγιά'          => '05-01',
+            'Δεκαπενταύγουστος'   => '08-15',
+            '28η Οκτωβρίου'       => '10-28',
+            'Χριστούγεννα'        => '12-25',
+            'Σύναξη της Θεοτόκου' => '12-26',
+
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+
+        $orthodox_easter = $this->orthodoxEaster($year);
```

Comment:
Could you use camelCase for PHP variable names?

Reply:
camelCase done! 

I think @dyanakiev from Bulgarian PR #132 can confirm that the current easternOrthodox() is not working

Reply:
My statement that it is not working it is not correct. 
I managed to reproduce correct easter date by adding ->setTimezone("Europe/Athens"); in the country.php
it is working correctly only if you add timezone in the $orthodoxEaster = $this->orthodoxEaster($year)->setTimezone("Europe/Athens");

Reply:
I notice that NorthMacedonia was passing the tests but for some reason they have timezone +1.

I will change this PR again. sorry for the mess.

Reply:
We are using orthodoxEaster in Greece. It is calculated differently. 

In my last commit seems to be working fine. the only downside is that orthodox easter needs setTimezone to work properly.

Yesterday I tried a diff approach but I changed the orthodoxEaster function. I took this back but I added setTimezone. 
Pls check my last commit

Reply:
Using startOfDay() on orthodoxEaster will result -1 day on some cases

Reply:
Try this in tinker: [https://gist.github.com/lowv-developer/5005f38930002e0a2620cfeb0984665e](url)

Reply:
I dont know whats going on. I see other eastern orthodox countries to pass all the tests. There is some kind of bug there but I cannot reproduce exactly. it has something to do with the time and timezones. I'm closing this.